tephra-hbase-compat-1.0/src/test/java/org/apache/tephra/hbase/coprocessor/TransactionVisibilityFilterTest.java

```diff
@@ -290,44 +290,52 @@ private void runFilteringTest(TxFilterFactory txFilterFactory,
   @Test
   public void testTTLFiltering() throws Exception {
     Map<byte[], Long> ttls = Maps.newTreeMap(Bytes.BYTES_COMPARATOR);
-    ttls.put(FAM, 10L);
-    ttls.put(FAM2, 30L);
+    ttls.put(FAM, 100L);
+    ttls.put(FAM2, 300L);
     ttls.put(FAM3, 0L);
 
+    // start a transaction to populate the in-progress list. It should not affect TTL calculations
+    txManager.startShort();
```

Comment:
abort this tx at the end of the test?

Reply:
Many other test cases don't complete the tx lifecycle by the end of their test cases.
I think this is fine because a new TransactionManager is used for each test case.